patches/net-dns-dns_transaction.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
+index 9bfd4cace384ca502716efa191eaa6ccfd6a54a8..9123e8cb94b566ef1a25cbc498dfc859e4165e73 100644
+--- a/net/dns/dns_transaction.cc
++++ b/net/dns/dns_transaction.cc
+@@ -1299,6 +1299,7 @@ class DnsTransactionImpl : public DnsTransaction,
+     DCHECK(secure_);
+ 
+     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
++    BRAVE_MAKE_HTTP_ATTEMPT
```

Comment:
This is to skip the DoH server we added for non-target TLDs (ex: non-crypto domains) when doing name resolution.